docs/user-guide.md

```diff
@@ -68,51 +68,21 @@ INFO Generated config file in assets directory: appliance-config.yaml
 
 ### Set `appliance-config`
 * Initially, the template will include comments about each option and will look as follows:
+* Check the [appliance-config](./appliance-config.md) details on how to set each parameter.
 
 ```yaml
-#
-# Note: This is a sample ApplianceConfig file showing
```

Comment:
I think we can keep these comments in the user-guide, no harm for having them also here. I've seen that some users just skim over the guide and don't follow references....

Reply:
ok. done.